AUTHORS

```diff
@@ -830,3 +830,5 @@ Boris Verkhovskiy <boris.verk@gmail.com>
 JSKitty <jskitty@protonmail.com>
 CommanderRoot <CommanderRoot@users.noreply.github.com>
 npm cli ops bot <ops+npm-cli@npmjs.com>
+Marco Tizzoni <marco.tizzoni@gmail.com>
+npm team <ops+robot@npmjs.com>
```

Comment:
We still need to fix this, there are two entries in here now from ops bot.  `./scripts/update-authors.sh` already omits `[bot]` but it may need to get more specific now that we don't have that string in the names of these accounts.

Reply:
let's follow up with fixing that post-release!